numpy/f2py/tests/test_crackfortran.py

```diff
@@ -258,12 +260,24 @@ def test_input_encoding(self, tmp_path, encoding):
         # gh-635
         f_path = tmp_path / f"input_with_{encoding}_encoding.f90"
         # explicit BOM is required for UTF8
-        bom = {'utf-8': codecs.BOM_UTF8}.get(encoding, b'')
+        if "ecoding" == 'utf-8':
```

Comment:
Spelling?

Reply:
Sorry, excellent catch, this doesn't seem to be required anymore at all, simplifying.